lib/route/route.js

```diff
@@ -237,30 +237,30 @@ const getRepeatableCount = (_id, params, userData) => {
 }
 
 const updateParams = (_id, params, direction) => {
-  // use $models ?
-  const models = []
-  const buildModels = (_id) => {
+  // use $namespaces ?
+  const namespaces = []
+  const buildNamespaces = (_id) => {
     const instance = getInstance(_id)
-    const model = instance.model
+    const namespace = instance.namespace
     const repeatable = instance.repeatable
-    if (model && repeatable) {
-      models.push(model)
+    if (namespace && repeatable) {
+      namespaces.push(namespace)
     }
     if (instance._parent) {
-      buildModels(instance._parent)
+      buildNamespaces(instance._parent)
     }
   }
-  buildModels(_id)
+  buildNamespaces(_id)
 
   params = params || {}
   const updatedParams = {}
-  models.forEach((model, index) => {
-    let paramValue = params[model]
-    if (params[model] === undefined) {
+  namespaces.forEach((namespace, index) => {
+    let paramValue = params[namespace]
+    if (params[namespace] === undefined) {
       if (direction === 'next') {
-        params[model] = 1
+        params[namespace] = 1
       } else if (direction === 'previous') {
-        params[model] = 1000000
+        params[namespace] = 1000000
```

Comment:
Unrelated but what's the 1000000 about?

Reply:
Just an arbitrarily high number that stands in place of infinity